beaver_store/lib/src/storage_service/gcloud_storage_service.dart

```diff
@@ -61,7 +64,13 @@ class BeaverBuild extends Model {
   String log;
 }
 
+// FIXME: In the current implementation, it's is not easy to obtain GCloudMixin
+// here because GCloudContext is created later then BeaverStore. Since
+// initializing the DatastoreDB only needs a project name and credentials, it
+// can be initialized in itself via _getDefaultDatastoreDB().
 class GCloudStorageService implements StorageService {
```

Comment:
I've looked and working on it, and I'm trying to initialize this through calling a kind of BeaverStore.init() on beaver_api and trigger_handler startup. Is there any batter way to initialize, just let me know.